README.md

```diff
@@ -42,22 +42,45 @@ The same goes for the component library:
 ```
 
 ### Develop or mockups
-To develop the DSO Toolkit using components and variants or create mockups of pages, forms or components
+**Important notice: We recently switched to Yarn**: It's possible that you're still looking at a topic branch that uses NPM:
+* If your repository contains `package-lock.json`, continue using `npm`.
+* If you see `yarn.lock`, that means you should start using `yarn`. See below for more information.
+
+**Everytime you switch to a revision (tag, branch, commit) that uses a different package manager (yarn or npm), you need to reinstall node_modules:**
+
+```
+npx rimraf node_modules
+npm install OR yarn install
```

Comment:
Running yarn with no command will run `yarn install`. Of vinden we het een nette notatie om toch `yarn install` te noemen (ik heb het dus zelf nog NOOIT ingetikt)

Reply:
Ik heb er een gewoonte van te maken om in documentatie en scripts de commando's altijd uit te typen. Voor iemand die niet bekend is met de techniek, of als het techniek is die niet dagelijks gebruikt wordt is het duidelijker.

Overigens vind ik het de grootste fout die Yarn heeft kunnen maken. Ik had liever dat `start` gelinked was aan alléén `yarn`: Ik wil mijn applicatie vaker starten dan installeren..

Reply:
Eens.